java/common/src/main/java/io/strimzi/common/ConfigUtil.java

```diff
@@ -2,6 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
+
```

Comment:
I don§t think we have an empty line here in most Strimzi files. So maybe you can here and elsewhere follow the same.